deploy-dotorg.sh

```diff
@@ -11,96 +11,198 @@ if [[ -z "$SVN_PASSWORD" ]] && [[ $1 != "preview" ]]; then
 	exit 1
 fi
 
+# These themes either:
+# 	are not (and not expected to be) deployed to wporg
+# 	have been deployed to wporg, however they are not expected to be updated
+# Note that all classic themes (those without a theme.json file) are ALSO ignored
+declare -a THEMES_TO_IGNORE=(
+        'aldente'
```

Comment:
It would be great if @beafialho @henriqueiamarino and @iamtakashi could check this list, just in case we're excluding any themes that should be deployed to dotorg.

I think the following are possible candidates: foam, cortado, grammerone, hall, raw.

Reply:
I THINK that I checked all of those and found that they hadn't been updated in > 2 years; that's why they are on the list.

Not ALL themes that haven't been updated in > 2 years are in that list... just themes that HAVE updates.

However the list gone through a few iterations so I'm not sure that I trust that any more.

Reply:
foam, cortado, grammerone, hall, and raw have all been added in the last year, and I can't see anything on the original "ready to launch" posts about them being dotcom-exclusives. But I'm also not 100% sure of the intention with these themes, which is why I thought it might be best to check in with the designers first.

Reply:
I think we can follow-up with feedback on the definitive list; for now we would benefit from focusing on getting the mechanism in, so that we can maintain that list from an ignore file.

Reply:
Sounds good 👍